test/server/server_test.cc

```diff
@@ -475,7 +475,8 @@ TEST_P(ServerInstanceImplTest, Stats) {
   options_.concurrency_ = 2;
   options_.hot_restart_epoch_ = 3;
   EXPECT_NO_THROW(initialize("test/server/test_data/server/empty_bootstrap.yaml"));
-  EXPECT_NE(nullptr, TestUtility::findCounter(stats_store_, "server.watchdog_miss"));
+  EXPECT_NE(nullptr, TestUtility::findCounter(stats_store_, "main_thread.watchdog_miss"));
+  EXPECT_NE(nullptr, TestUtility::findCounter(stats_store_, "workers.watchdog_miss"));
```

Comment:
Existing monitoring for "server.watchdog_miss" will break for users that are configured using the current single watchdog parameter.

Is this acceptable?  Worth mentioning i release notes?

Reply:
I've added it to the release notes. 

From a lot of the discussion with @akonradi, trying to maintain full backwards compatibility would end up with significant complexity within the system. Furthermore, as you pointed out in https://github.com/envoyproxy/envoy/pull/13015#discussion_r485825780, there will have to be some breakage at some point (either in the future or now). 

Given those, I went for simplicity at the cost of minor behavior changes now.

Reply:
Great.

Reply:
Yeah we don't have a stats deprecation policy right now, so it's ad hoc and "best judgement" depending on how many people we think we will break. I think this is OK.